src/utils/helper.ts

```diff
@@ -55,6 +55,8 @@ const getSelfHostedLogo = (ticker: string) => {
   switch (ticker) {
     case "DYDX":
       return "/assets/dydx.svg";
+    case "PALM":
```

Comment:
Side note: would be simpler to name the files after the tickers, so we don't need to manually maintain this function (just use `'/assets/${ticker}.svg'` instead)

Reply:
Good point, will correct !